### PR TITLE
Query fixes

### PR DESCRIPTION
**Pull request by jjchico**
Simplifies number of users querying. Also correct printf 0-padding format to better reflect the original intention.
May solve issue #65

### CODE REVIEW

**Comment by jjchico** [comment]
Thanks for the merge.

**Comment by sandain** [comment]
Thanks for the patches!

File: mscs [hunk 1]
```diff
@@ -1965,7 +1979,7 @@ case "$1" in
     if serverRunning "$2"; then
       # If the world server has users logged in, announce that the world is
       # being deleted.
-      if [ $(printf "%d" $(queryStatus $2 | cut -f6)) -gt 0 ]; then
+      if [ $(queryNumUsers $WORLD) -gt 0 ]; then
```
Comment (sandain) [inline mscs]: Hmm, I didn't look very closely before accepting this patch.  Shouldn't this line be:
```
      if [ $(queryNumUsers $2) -gt 0 ]; then
```

File: mscs [hunk 2]
```diff
@@ -1991,7 +2005,7 @@ case "$1" in
     if serverRunning "$2"; then
       # If the world server has users logged in, announce that the world is
       # being disabled.
-      if [ $(printf "%d" $(queryStatus $2 | cut -f6)) -gt 0 ]; then
+      if [ $(queryNumUsers $WORLD) -gt 0 ]; then
```
Comment (sandain) [inline mscs]: same here

**Comment by sandain** [comment]
I just noticed your new pull request.  This issue is fixed there.

**Pull request merged**